src/idiom286.fuzion

```diff
@@ -0,0 +1,5 @@
+_ := for
```

Comment:
You should not need the  `_:=` here since `say` result in `unit` and thus the loops result is `unit`

Reply:
I've noticed that by now. I've had an error with a for loop once and then thought it would always be necessary if the the result is not used.